robotoff/workers/tasks/import_image.py

```diff
@@ -917,6 +926,12 @@ def generate_ingredient_prediction_data(
         # available
         if entity["lang"]:
             lang_id = entity["lang"]["lang"]
+            # Skip if the language code is not a valid 2-letter code
```

Comment:
Can you add further explanation of why this is needed (=because Product Opener only supports ISO-639-1 codes and not ISO-639-3)?